L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -277,6 +277,17 @@ namespace trklet {
     void setStripLength_PS(double stripLength_PS) { stripLength_PS_ = stripLength_PS; }
     void setStripLength_2S(double stripLength_2S) { stripLength_2S_ = stripLength_2S; }
 
+    int numbins() const { return numbins_; }
```

Comment:
variable name here gives no clue it is related to Pt bins of DR.

Reply:
I have renamed this to numrinvbins.